env.yml

```diff
@@ -1,12 +1,14 @@
 name: payu
 channels:
   - accessnri
-  - coecms
   - conda-forge
+  - coecms
```

Comment:
We'd want the `coecms` channel to be in the same position it was at the start of this PR - I think the channels list is in priority order, and we'd want to check `coecms` before the wider `conda-forge` channel, right?

Reply:
Should priority go to conda-forge as we want to try move away using packages on `coecms`?

Reply:
I've changed it back to the same position in [4d8f08e](https://github.com/ACCESS-NRI/payu-condaenv/pull/35/commits/4d8f08eca2bac59268d4b1c4a41f3838ea419971)